### ISSUE TITLE
[Bug Report][2.3.7] Hint/Description messages of an Input field are not read by Screen readers

### ISSUE THREAD

**Issue by NivyaPallempati** [vuetifyjs/vuetify#13874]
### Environment
**Vuetify Version:** 2.3.7
**Vue Version:** 2.6.11
**Browsers:** Chrome 91.0.4472.124
**OS:** Windows 10

### Steps to reproduce
Use Screen reader and try to access the text fields. Though value and labels are interpreted, hint message is ignored. If any error occurs based on rules, that is read(error message with role:alert) but not general hint messages.

### Expected Behavior
Hint messages provided to an input field should be interpreted by a Screen reader.

### Actual Behavior
Any message that is given as a hint should be interpreted by any Screen reader tool.

### Reproduction Link
<a href="https://codepen.io/NivyaPallempati/pen/KKmpdoG?editors=101" target="_blank">https://codepen.io/NivyaPallempati/pen/KKmpdoG?editors=101</a>

### Other comments
I have tested this with NVDA. The issue is reproduced with the latest vuetify version too.
